doc/reference/power_management/index.rst

```diff
@@ -330,20 +325,17 @@ Device Set Power State
 
 .. code-block:: c
 
-   int pm_device_state_set(const struct device *dev, uint32_t device_power_state, pm_device_cb cb, void *arg);
+   int pm_device_state_set(const struct device *dev, enum pm_device_state *state);
```

Comment:
The signature is wrong. State is not a pointer

Reply:
fixed